litmus-portal/backend/auth/pkg/types/const.go

```diff
@@ -5,7 +5,7 @@ import "time"
 // define the type of authorization request
 const (
 	RedirectURI                string = "http://localhost:3000/"
-	DefaultDBServerURL         string = "mongodb://mongo-service.default.svc.cluster.local:27017"
+	DefaultDBServerURL         string = "mongodb://mongo-service.litmus.svc.cluster.local:27017"
```

Comment:
If all the components are deployed in the same namespace, I think we can just use the name of the service and not the full url. `eg. "mongodb://mongo-service:27017"` or something similar to this. Can you try this.